cost/azure/instance_rightsizing/azure_instance_rightsizing.pt

```diff
@@ -0,0 +1,732 @@
+name "Azure Rightsize Virtual Machine Instances"
+rs_pt_ver 20180301
+type "policy"
+short_description "Rightsizes Azure virtual machine instances based on CPU and Memory thresholds.  See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/instance_rightsizing) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
use:  Finds Azure Virtual Machine instances based on CPU and Memory thresholds and resizes them after approval.